.github/workflows/ci.yml

```diff
@@ -121,7 +127,7 @@ jobs:
           target: x86_64-unknown-linux-musl
 
       - name: Build Connector
-        run: cdk build -p http-sink
+        run: cdk build -p http-sink --target x86_64-unknown-linux-musl
```

Comment:
seems to already be targeting musl, is this needed?

Reply:
not needed but maybe it is better to be explicit?